### ISSUE TITLE
Game modes

### ISSUE THREAD

**Issue by mlutze** [mlutze/flixball#56]
Now that we have some basic AIs, I think we should consider alternative game modes. We can probably take ideas from classic shooters:
- Capture the Flag
- Area control
- Kill everyone
- Zombie mode

These will require a few more game mechanics.
- Score tracking
- Teams
- Respawning
- Ending the game

And possibly:
- Health
- Fog of war
- Team communication

**Comment by JonathanStarup** [comment]
I think time tracking will also be helpful.

**Comment by JonathanStarup** [comment]
Maybe `Kill everyone` is the simplest and could be part of the initial release or version 1.0, whatever it will be called.

**Comment by mlutze** [comment]
> Maybe `Kill everyone` is the simplest and could be part of the initial release or version 1.0, whatever it will be called.

Agreed. If everyone just has one life then this boils down to what we already have, but just detecting if there's only one player left.
